screens/RandomQuote.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from "react";
+import {
+	View,
+	Text,
+	ActivityIndicator,
+	Vibration,
+	StyleSheet,
+} from "react-native";
+import { Accelerometer } from "expo-sensors";
+import { LinearGradient } from "expo-linear-gradient";
```

Comment:
unfortunately i cannot see the app in my mobile, but i would love to see this linear gradient effect